src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+type FigureShape = 'triangle' | 'circle' | 'rectangle';
+type FigureColor = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: FigureShape;
+  color: FigureColor;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: FigureShape = 'triangle';
 
-}
+  constructor(
+    public color: FigureColor,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const isUnValidSides = this.a < 0 || this.b < 0 || this.c < 0;
```

Comment:
better to start your booleans with positive
```suggestion
    const areSidesValid = this.a > 0 && this.b > 0 && this.c > 0;
```